src/main/kotlin/espressoLogcat/LogcatRuntime.kt

```diff
@@ -0,0 +1,59 @@
+package espressoLogcat
+
+import java.io.BufferedReader
+import java.io.InputStreamReader
+
+interface ILogcatExecutor {
```

Comment:
A more Android-y way to name this would be calling the interface `LogcatExecutor` and then the default implementation `LogcatExecutorImpl` or something. the `I` prefix for interfaces is more of a C# thing usually

Reply:
I blame Paul for giving me bad Java-y advice. I'll update this.

Reply:
Fixed in bb55ef4